presto-docs/src/main/sphinx/release/release-0.202.rst

```diff
@@ -0,0 +1,78 @@
+=============
+Release 0.202
+=============
+
+General Changes
+---------------
+
+* Fix correctness issue for queries involving aggregations over the result of an outer join
+  (`issue #10592 <https://github.com/prestodb/presto/issues/10592>`_).
+* Fix :func:`map` to fail on duplicate keys, instead of producing a corrupted map silently.
+* Fix OOM in bucketed execution by scheduling new splits on the same worker as
+  the recently finished one.
+* Fix query failure when performing a ``GROUP BY`` on ``JSON`` and ``IPADDRESS`` types.
+* Fix :func:`map_from_entries` to throw ``INVALID_FUNCTION_ARGUMENT`` when input
+  array contains a ``null`` entry.
+* Fix correctness issue :func:`line_locate_point`, :func:`ST_IsValid`, and :func:`geometry_invalid_reason`
+  where they were returning values outside of the expected range.
+* Fix failure in :func:`geometry_to_bing_tiles` and :func:`ST_NumPoints` functions when
```

Comment:
This seems rather vague. Does this mean they don't work at all geometry collections (but they work on other things)?